posthog/backup.py

```diff
@@ -0,0 +1,15 @@
+from functools import lru_cache
```

Comment:
This file should not be on the top level since it is ee-specific. Instead:

- Move it under ee/
- update the return value in _system_status in the existing conditional.